### PR TITLE
support to get node IP

### PR DESCRIPTION
**Pull request by John-Lin**

### CODE REVIEW

File: src/kubernetes/nodes_test.go [hunk 1]
```diff
@@ -1,56 +1,86 @@
...
+	_, err = suite.fakeclient.CoreV1().Nodes().Create(&node)
 	assert.NoError(t, err)
 
-	nodes, err := GetNodes(clientset)
+	nodes, err := suite.kubectl.GetNodes()
 	assert.NoError(t, err)
 	assert.Equal(t, 2, len(nodes))
 }
+
+func (suite *KubeCtlTestSuite) TestGetNodeExternalIP(t *testing.T) {
```
Comment (hwchiu) [inline src/kubernetes/nodes_test.go]: Add the test case to get the non-existed node name

**Comment by John-Lin** [comment]
@hwchiu done

**Comment by hwchiu** [comment]
why the coverage from 80% to 0%? 

src/kubernetes/nodes.go | 0% <0%> (-80%)

**Comment by John-Lin** [comment]
fixed

**Pull request merged**